service/openapi.yaml

```
@@ -360,9 +360,11 @@ components:
         enabled:
           type: boolean
           description: set to true if the remote flag management system is supporting polling
-        minPollingInterval:
+        minPollingIntervalInMs:
```

```suggestion
        minPollingIntervalMs:
```

This is another option but I could go either way as long as we stay consistent.

---

I think `minPollingIntervalMs` is better